daemon/response.go

```diff
@@ -428,23 +428,20 @@ var (
 
 // SnapNotFound is an error responder used when an operation is
 // requested on a snap that doesn't exist.
-func SnapNotFound(snapName string, err error) Response {
-	return &resp{
-		Type: ResponseTypeError,
-		Result: &errorResult{
-			Message: err.Error(),
-			Kind:    client.ErrorKindSnapNotFound,
-			Value:   snapName,
-		},
-		Status: 404,
+func SnapNotFound(snapName string, err error) *apiError {
+	return &apiError{
```

Comment:
This is very nice especially when comparing to the previous way of writing this.